mmcv/image/__init__.py

```diff
@@ -2,11 +2,11 @@
 from .transforms import (bgr2gray, gray2bgr, bgr2rgb, rgb2bgr, bgr2hsv,
                          hsv2bgr, imflip, imrotate, imcrop, impad,
                          impad_to_multiple, imnormalize, imdenormalize,
-                         imresize, imresize_like, imrescale)
+                         imresize, imresize_like, imrescale, iminvert)
```

Comment:
It's better to put methods from the same module together (`iminvert` and `xxx2xxx`).